src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -18,12 +17,52 @@ class AssetsTable extends React.Component {
       modalOpen: false,
       assetToDelete: {},
       elementToFocusOnModalClose: {},
+      tableColumns: [
+        // keyed object? or just search through the object when I need it? performance?
```

Comment:
Performance-wise, this is O(n) for linear search vs O(1) for hash lookup in a keyed object right?

I doubt it matters much when n is 4, but if it's not difficult to do I think the keyed object is the "right" decision here.